### ISSUE TITLE
Can't run gevent-get-input.py on Windows (ptpython and IPython have the same issue)

### ISSUE THREAD

**Issue by maryush** [jonathanslenders/python-prompt-toolkit#367]
Hi, i install ptpython from pip, but when i try to start i got error like:

```
Traceback (most recent call last):
  File "c:\python27\lib\runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "c:\python27\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "c:\python27\lib\site-packages\ptpython\__main__.py", line 7, in <module>
    run()
  File "c:\python27\lib\site-packages\ptpython\entry_points\run_ptpython.py", line 71, in run
    title='Python REPL (ptpython)')
  File "c:\python27\lib\site-packages\ptpython\repl.py", line 309, in embed
    cli.run()
  File "c:\python27\lib\site-packages\prompt_toolkit\interface.py", line 389, in run
    self.eventloop.run(self.input, self.create_eventloop_callbacks())
  File "c:\python27\lib\site-packages\prompt_toolkit\eventloop\win32.py", line 66, in run
    self._inputhook_context.call_inputhook(ready)
  File "c:\python27\lib\site-packages\prompt_toolkit\eventloop\inputhook.py", line 82, in call_inputhook
    select_fds([self._r], timeout=None)
  File "c:\python27\lib\site-packages\prompt_toolkit\eventloop\select.py", line 49, in select_fds
    result = _select(read_fds, timeout)
  File "c:\python27\lib\site-packages\prompt_toolkit\eventloop\select.py", line 99, in _select
    return select.select(read_fds, [], [], timeout)[0]
select.error: (10038, 'Pr\xf3bowano przeprowadzi\xe6 pr\xf3b\xea na obiekcie, kt\xf3ry nie jest gniazdem')
```

I spend some time to tested it and found that one prompt toolkit example not working to with the same errors:

```
c:\tmp\examples>gevent-get-input.py
Give me some input:
Traceback (most recent call last):
  File "C:\tmp\examples\gevent-get-input.py", line 21, in <module>
    answer = prompt('Give me some input: ', eventloop=eventloop)
  File "c:\Python27\lib\site-packages\prompt_toolkit\shortcuts.py", line 540, in prompt
    eventloop=eventloop)
  File "c:\Python27\lib\site-packages\prompt_toolkit\shortcuts.py", line 621, in run_application
    result = cli.run(reset_current_buffer=False)
  File "c:\Python27\lib\site-packages\prompt_toolkit\interface.py", line 389, in run
    self.eventloop.run(self.input, self.create_eventloop_callbacks())
  File "c:\Python27\lib\site-packages\prompt_toolkit\eventloop\win32.py", line 66, in run
    self._inputhook_context.call_inputhook(ready)
  File "c:\Python27\lib\site-packages\prompt_toolkit\eventloop\inputhook.py", line 82, in call_inputhook
    select_fds([self._r], timeout=None)
  File "c:\Python27\lib\site-packages\prompt_toolkit\eventloop\select.py", line 49, in select_fds
    result = _select(read_fds, timeout)
  File "c:\Python27\lib\site-packages\prompt_toolkit\eventloop\select.py", line 99, in _select
    return select.select(read_fds, [], [], timeout)[0]
  File "c:\Python27\lib\site-packages\gevent\select.py", line 67, in select
    raise error(*ex.args)
select.error: 3 is not a socket (files are not supported)
```

Any idea what's wrong? How to fix it? I try to run it on Windows 10 Pro x64, Python 2.7.10 from python-xy package.

**Comment by takluyver** [comment]
Can you try with my PR #368 ?

**Comment by jonathanslenders** [comment]
Hi @maryush,

This was a bug that slipped into the latest prompt_toolkit release.
It has been fixed now in 1.0.5. Can you please do ``pip install -U prompt_toolkit`` and try again?
Excuse me for the inconvenience!

**Comment by maryush** [comment]
After update working excellent now - thx :)

He who makes no mistakes makes nothing ;)
